trillian/ctfe/cert_checker_test.go

```diff
@@ -174,9 +175,31 @@ func TestValidateChain(t *testing.T) {
 			chain:   pemFileToDERChain(t, "../testdata/subleaf.misordered.chain"),
 			wantErr: true,
 		},
+		{
+			desc:    "reject-ext-id",
+			chain:   pemsToDERChain(t, []string{testonly.LeafSignedByFakeIntermediateCertPEM, testonly.FakeIntermediateCertPEM}),
+			wantErr: true,
+			modifyOpts: func(v *CertValidationOpts) {
+				// reject SubjectKeyIdentifier extension
+				v.rejectExtIds = []asn1.ObjectIdentifier{[]int{2, 5, 29, 14}}
+			},
+		},
+		{
+			desc:    "reject-ext-id-precert",
+			chain:   pemsToDERChain(t, []string{testonly.PrecertPEMValid, testonly.FakeIntermediateCertPEM}),
+			wantErr: true,
+			modifyOpts: func(v *CertValidationOpts) {
+				// reject SubjectKeyIdentifier extension
+				v.rejectExtIds = []asn1.ObjectIdentifier{[]int{2, 5, 29, 14}}
+			},
+		},
```

Comment:
Also test that ValidateChain succeeds when rejectExtIds is set but the cert doesn't have any matching extensions.

Reply:
Good call, done